src/main/features/claim/helpers/featuresBuilder.ts

```diff
@@ -1,46 +1,75 @@
-import { FeatureToggles } from 'utils/featureToggles'
-import { FeatureTogglesClient } from 'shared/clients/featureTogglesClient'
 import { ClaimStoreClient } from 'claims/claimStoreClient'
 import { User } from 'idam/user'
-
-const claimStoreClient: ClaimStoreClient = new ClaimStoreClient()
-const featureTogglesClient: FeatureTogglesClient = new FeatureTogglesClient()
+import { LaunchDarklyClient } from 'shared/clients/launchDarklyClient'
+import * as config from 'config'
 
 export class FeaturesBuilder {
   static readonly MEDIATION_PILOT_AMOUNT = 500
   static readonly LA_PILOT_THRESHOLD = 300
   static readonly JUDGE_PILOT_THRESHOLD = 10000
   static readonly ONLINE_DQ_THRESHOLD = 10000
 
-  static async features (amount: number, user: User): Promise<string> {
-    const roles: string[] = await claimStoreClient.retrieveUserRoles(user)
+  readonly claimStoreClient: ClaimStoreClient
+  readonly launchDarklyClient: LaunchDarklyClient
 
-    let features = ''
-    if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_admissions')) {
-      features = 'admissions'
-    }
+  constructor (claimStoreClient: ClaimStoreClient, launchDarklyClient: LaunchDarklyClient) {
+    this.claimStoreClient = claimStoreClient
+    this.launchDarklyClient = launchDarklyClient
+  }
 
-    if (amount <= this.MEDIATION_PILOT_AMOUNT) {
-      if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_mediation_pilot')) {
-        features += features === '' ? 'mediationPilot' : ', mediationPilot'
-      }
-    }
+  async features (amount: number, user: User): Promise<string> {
+    const roles: string[] = await this.claimStoreClient.retrieveUserRoles(user)
 
-    if (amount <= this.LA_PILOT_THRESHOLD) {
-      if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_legal_advisor')) {
-        features += features === '' ? 'LAPilotEligible' : ', LAPilotEligible'
-      }
-    } else if (amount <= this.JUDGE_PILOT_THRESHOLD) {
```

Comment:
This `else` was missed and so this PR introduced bug [ROC-8056](https://tools.hmcts.net/jira/browse/ROC-8056)